src/mingw.thread.h

```diff
@@ -64,7 +64,8 @@ class thread
             using Call = decltype( std::bind( f, args... ) );
             Call *call = new Call( std::bind( f, args... ) );
             mHandle = ( HANDLE )_beginthreadex( NULL, 0, threadfunc<Call>,
-                                                ( LPVOID )call, 0, ( unsigned * ) & ( mThreadId.mId ) );
+                                                static_cast<LPVOID>( call ),
+                                                0, ( unsigned * ) & ( mThreadId.mId ) );
```

Comment:
What about this `unsigned *`, is this not also a cast?

Reply:
Yes, there are maybe a few more casts - which I could miss using web UI.